### ISSUE TITLE
Commas attached to preceding token

### ISSUE THREAD

**Issue by patrickpilch** [shyiko/ktlint#173]
[Android's Kotlin style guide's "Where to break"](https://android.github.io/kotlin-guides/style.html#where-to-break) number <span>#</span>4 states:

> A comma (,) stays attached to the token that precedes it.

It seems ktlint does not enforce this, so it allows potentially unwanted formatting like:
```kotlin
some.method(1 , 2)
```
Or
```kotlin
fun testMethod(
    arg1: Int ,
    arg2: Int
    ,

    arg3: Int
) = Unit
```

Version tested: 0.19.0

**Comment by shyiko** [comment]
Thanks Patrick. Yeah, [SpacingAroundCommaRule](https://github.com/shyiko/ktlint/blob/master/ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/SpacingAroundCommaRule.kt) is way to lax. I'll keep this ticket in mind when scoping for the next release.

**Comment by shyiko** [comment]
Fixed in [0.20.0](https://github.com/shyiko/ktlint/releases/tag/0.20.0).
